appmap/unittest.py

```diff
@@ -4,4 +4,3 @@
 
 if not Env.current.is_appmap_repo and Env.current.enables("unittest"):
     logger.debug("Test recording is enabled (unittest)")
-    import _appmap.unittest  # pyright: ignore pylint: disable=unused-import
```

Comment:
This import needs to be retained, so it should be marked with `# noqa: F401` rather than deleted.

Reply:
Fixed